index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Capstone</title>
+</head>
+<body>
+    <section class="container">
+        <header>
+            <nav>
+                <div class="desktop-nav">
+                    <img class="facebook-icon" src="./images/base/facebook.png" alt="">
+                    <img class="twitter-icon" src="./images/base/twitter.png" alt="">
+                    <ul>
+                        <li><a href="">English</a></li>
+                        <li><a href="">My Page</a></li>
+                        <li><a href="">Logout</a></li>
+                    </ul>
+                </div>
+                <div class="desktop-navbar">
+                    <a class="logo" href="">HUDSON</a> 
+                    <ul class="desktop-nav-items">
+                        <li class="nav-items"><a href="about.html">About</a></li>
+                        <li class="nav-items"><a href="">Services</a></li>
+                        <li class="nav-items"><a href="">Sponsors</a></li>
+                    </ul>
+                </div>
+                <div class="menu-bar">
+                    <div class="burger" id="btn">
+                        <span class="bar" id="btn1"></span>
+                        <span class="bar"></span>
+                        <span class="bar" id="btn2"></span>
+                    </div>
+                </div>
+            </nav>

```

Comment:
- [ ] [REQUIRED] Good job so far! Please set correct color for background of `First nav menu` and add all `menu options` to `second  nav menu` which provided [Behance Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

![image](https://github.com/brighton0102/HTML-CSS-JavaScript-Capstone-Project/assets/63915024/d1683901-c325-44f3-ae1b-b3456c501dbf)